### ISSUE TITLE
dpi settings in IPdfOptions

### ISSUE THREAD

**Issue by vatsenko** [Novaleaf/phantomjscloud-node#5]
I found that new option implemented in API - DPI, however, IPdfOptions don't contain this key in io-data-types.ts

**Comment by jasonswearingen** [comment]
Hi, it's not documented because unfortunately it's quite buggy (due to a bug in the underlying PhantomJs software)  so we decided against advertising it.   if you feel that it's actually useful, please do let me know though :)

**Comment by vatsenko** [comment]
This may be a useful option.

By default rendered PDF files have 72 DPI. (mistyped in Documentation - 150 DPI)
I use pdf stream for label printers that connect through Google Cloud Print and some printers (Zebra for example) use the physical size of PDF for printing, and the option “Fit to page” is not available through API.
Label printer has the native resolution - 300 DPI and this is not multiple 72 that's why I got artefact on the edge of barcode lines, and barcode became unreadable.

Actually, we switch to Brother due to this limitation - brother 720 has fit to page option.

As I get this is just matter of pdf Page size in inch …

**Comment by jasonswearingen** [comment]
hi, sorry i think we won't fix this, at least not until either phatomjs fixes pdf sizing (likely never) or until we move to a new rendering system (in the next few months hopefully)  so closing the issue.
